pkg/neutronapi/deployment.go

```diff
@@ -43,15 +43,15 @@ func Deployment(
 ) (*appsv1.Deployment, error) {
 	livenessProbe := &corev1.Probe{
 		// TODO might need tuning
```

Comment:
Hi, sure. Just so I understand. We want to have separated probes for both the API and the HTTP container. And, also remove the readiness probe from API since it's not user facing ? Is that correct ?

Reply:
Ok so current failures are due to liveness probe failure and that happens as we bind to localhost in neutron.conf bind_host = localhost while kubelet checks on podIP.
We could get it work by using bind_host = <podIP> , and getting podIP require extra work like using env.DownwardAPI and some pre injection(likely with an init container) of podIP from ENV Vars into mounted neutron and httpd configurations.
@booxter @slawqo wdyt is it worth to handle the probes separately as suggested above? probably not in this patch but as separate effort and limit the scope of this patch just to "increasing probe timeouts and removal of readiness probe from the api pod + TODO comment about possible improvements"

Reply:
Thanks Yatin. Ok, so let's adjust the probes in this patch so we can mitigate the immediate issue of the services being restarted no a busy environment. And, as a follow-up we can continue the research on how to split these probes to make it more specific for each container (httpd and neutron-api).